test/BF16/brgemm-tpp.mlir

```diff
@@ -1,17 +1,7 @@
-// RUN: tpp-opt -transform-dialect-interpreter -one-shot-bufferize="bufferize-function-boundaries allow-return-allocs function-boundary-type-conversion=identity-layout-map" --convert-vnni-to-tpp %s | FileCheck %s
-
-transform.sequence failures(propagate) {
- ^bb0(%arg1: !pdl.operation):
-  %0 = transform.structured.match ops{["linalg.batch_reduce_matmul"]} in %arg1
-    transform.structured.pack %0 { use_vnni=true, blocking_factors = [2] }
-}
+// RUN: tpp-opt -pack-vnni="block-factors=2" -generalize-tensor-pack-unpack -empty-tensor-to-alloc-tensor -one-shot-bufferize="bufferize-function-boundaries allow-return-allocs function-boundary-type-conversion=identity-layout-map" --convert-vnni-to-tpp %s | FileCheck %s
```

Comment:
We really should stop using `-empty-tensor-to-alloc-tensor` in all our tests. We need to start building the tests correctly, or fixing them to be correct. This generic allocation passes create more problems than they solve.

Reply:
I can follow-up on this.